### PR TITLE
Create ui-c8y-1021-0-0-fieldbus-write-register.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by bozhidardedov

Ticket: [DM-4002](https://cumulocity.atlassian.net/browse/DM-4002)
Original PR: [7275](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7275)

[DM-4002]: https://cumulocity.atlassian.net/browse/DM-4002?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1021-0-0-fieldbus-write-register.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fieldbus write register
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1021-0-0-fieldbus-write-register.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Added support for writing registers on fieldbus devices
```

File: content/change-logs/device-management/ui-c8y-1021-0-0-fieldbus-write-register.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-4002
+version: 1021.0.0
+---
+Fieldbus write register
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1021-0-0-fieldbus-write-register.md:17]: The Claude AI suggested to use the following body:
```suggestion
To enable more advanced interactions with fieldbus devices connected to Cumulocity IoT, support for writing registers on these devices has been added. With this change, it is now possible to not only read values from fieldbus devices, but also write configuration or command registers on them directly from Cumulocity IoT. This greatly expands the capabilities when integrating machines and sensors connected via fieldbus protocols into the IoT platform, allowing for remote configuration and control in addition to monitoring.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by reey** [comment]
Closing as already covered in: https://github.com/Cumulocity-IoT/c8y-docs/pull/2561

**Pull request closed without merging**